lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -0,0 +1,104 @@
+require 'ipaddr'
+require 'pathname'
+
+module LinuxAdmin
+  class NetworkInterfaceRH < NetworkInterface
```

Comment:
What is the `RH` in `NetworkInterfaceRH`?  Does that stand for Red Hat (the company) or RHEL (the OS)?  If the latter, then I would rename to `NetworkInterfaceRHEL`.